docs/userguide/parameters.md

```diff
@@ -127,11 +127,38 @@ SubstanceParameter(
 )
 ```
 
-The ``encoding`` option defines what kind of descriptors are calculated:
-* ``MORDRED``: 2D descriptors from the [Mordred package](https://mordred-descriptor.github.io/documentation/master/).
-  Since the original package is now unmaintained, baybe requires the community replacement [mordredcommunity](https://github.com/JacksonBurns/mordred-community)
-* ``RDKIT``: 2D descriptors from the [RDKit package](https://www.rdkit.org/)
-* ``MORGAN_FP``: Morgan fingerprints calculated with RDKit (1024 bits, radius 4)
+The ``encoding`` option defines what kind of descriptors are calculated.
+All descriptors are calculated using 
+[scikit-fingerprints package](https://github.com/scikit-fingerprints/scikit-fingerprints/).
+Any fingerprint class from `scikit-fingerprints` can be used as an input parameter for chemical encoding.
+The fingerprint class names should be passed in all upper case and without the `Fingeprint` suffix,
+e.g. use alias `MORDRED` for `MordredFingerprint` class.
+Here are examples of a few popular fingerprints:
+* ``ECFP``: Extended Connectivity FingerPrint, 
+which is a circular topological fingerprint similar to Morgan fingerprint.
+* ``MORDRED``: Chemical descriptor based fingerprint.
+* ``RDKIT``: The RDKit fingerprint, which is based on hashing of molecular sub-graphs.
+
+You can adjust fingerprint computation with parameters for `Fingerprint` classes from `scikit-fingerprints`.
+These can be specified via the `kwargs_fingerprint` in the `SubstanceParameter` class.
+
+```python
+from baybe.parameters import SubstanceParameter
+
+SubstanceParameter(
+    name="Solvent",
+    data={
+        "Water": "O",
+        "1-Octanol": "CCCCCCCCO",
+        "Toluene": "CC1=CC=CC=C1",
+    },
+    encoding="ECFP", 
+    kwargs_fingerprint = {
+        "n_jobs" : 2,  # Parallelize fingerprint computation across input molecules
```

Comment:
very nice addition to the UG, but I would not sue `n_jobs` as example because I think ur implementaiton only calls teh function on asingle molecule anyway + the `radius` is a muchmore commonly changed parameter, so why not use `radius` ?

Reply:
removed - though n_jobs may be useful to show, but indeed does not make sense here

Reply:
ive added the radius